### ISSUE TITLE
[bug] bactopia search does not create output directory if it doesn't exist

### ISSUE THREAD

**Issue by rpetit3** [bactopia/bactopia#135]
**Description**
At the moment, bactopia search allows you to specify an output directory. But, if the directory doesn't exist, bactopia search doesn't create it.

**Steps to Reproduce**
```
bactopia search PRJNA480016 --limit 20 --outdir test-search
Traceback (most recent call last):
  File "/local/home/rpetit/miniconda3/envs/bactopia/bin/bactopia-search.py", line 303, in <module>
    with open(results_file, 'w') as output_fh:
FileNotFoundError: [Errno 2] No such file or directory: 'test-search/ena-results.txt'
```

**Expected Behavior**
The directory should be created, then continue as normal

**Execution Environment**

- Bactopia Version: v1.4.9
- OS: Debian
- Environment: Conda

**Additional Information**
Tagging @tauqeer9

**Comment by rpetit3** [comment]
Fixed: https://github.com/bactopia/bactopia/commit/d850bd151315ba0cfa8807e137fa230c50df9b7a

**Comment by rpetit3** [comment]
Fixed as of v1.4.10
